### PR TITLE
Cargo Piano Orders

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
Added 4 "piano" like instruments to cargo.

Piano
Upright Piano
Church Organ
Minimoog

## Why / Balance
No way to get them unless you buy ships that has them.

## Technical details
.yml

## Media
- [X] this PR does not require an ingame showcase

## Breaking changes
N/A

**Changelog**
:cl: dvir01
- add: NT listened to people needs, and now selling 4 piano like instruments in cargo

### CODE REVIEW

**Comment by McBosserson** [comment]
I think the changelog is too wordy. Instead of listing everything out, maybe put "a new collection of large keyed instruments". People can always refer to the github or crate description for details.

**Comment by dvir001** [comment]
> I think the changelog is too wordy. Instead of listing everything out, maybe put "a new collection of large keyed instruments". People can always refer to the github or crate description for details.

Its not crate, its 4 items.

**Review by Cheackraze — APPROVED** [review]
